.github/ISSUE_TEMPLATE/flaking-test.md

```diff
@@ -1,5 +1,7 @@
 ---
-name: Flaking Test about: Report flaky tests or jobs in TiCDC CI labels: component/test
+name: Flaking Test 
```

Comment:
may be we can add a cute emoj here~

Reply:
I would love to add a 😈 because it's so much and hard to fix.

Reply:
or 😢 It can express our feelings about it.